tests/test_CSVTimeSeriesReader.py

```diff
@@ -460,15 +460,19 @@ def test_altitude_filter_2(self):
                 "country": "NO",
                 "standard_deviation": "NaN",
                 "flag": "0",
-            }
+            },
         ) as ts:
             self.assertEqual(len(ts.stations()), 1)
 
     def test_altitude_filter_3(self):
         engines = pyaro.list_timeseries_engines()
         with engines["csv_timeseries"].open(
             filename=self.elevation_file,
-            filters=[pyaro.timeseries.filters.get("altitude", min_altitude=150, max_altitude=250)],
+            filters=[
+                pyaro.timeseries.filters.get(
+                    "altitude", min_altitude=150, max_altitude=250
+                )
+            ],
```

Comment:
Why is the formatting changed here? Different black settings? Different formatter?

Reply:
Might be a different formatter? I am using `black==23.9.1`, consistent with `.pre-commit-config.yaml`. I could add a linting step to the CI

Reply:
My black setup also changes the lines. No idea how @thorbjoernl managed to commit the original code without formatting?

Reply:
Keep the change, this is fine.